openquakeplatform/openquakeplatform/vulnerability/static/vulnerability/js/capacity_curves.js

```diff
@@ -59,7 +68,9 @@ catch(e) {
 
 try {
     var generalComments = gl.fields.general_comments;
-    generalComments = generalComments.replace(/\r\n/g , "<br>");
+    generalComments = generalComments.replace(/</g, '&lt;');
+    generalComments = generalComments.replace(/>/g, '&gt;');
+    generalComments = generalComments.replace(/\r/g , "<br>");
 }
```

Comment:
Please add a utility function to do it and reuse in any case you need it.
Take a look here: http://stackoverflow.com/questions/1787322/htmlspecialchars-equivalent-in-javascript